view-creator-framework/src/main/java/org/hypertrace/core/viewcreator/pinot/PinotTableSpec.java

```diff
@@ -36,6 +36,7 @@ public class PinotTableSpec {
 
   // Segments config
   private int numReplicas;
+  @Optional private String replicasPerPartition = "1";
```

Comment:
by default, it  should be set with same value as `numReplicas`?

Reply:
These are 2 independent configurations.
Intentionally decoupled this configuration.